org.thonny.Thonny.yaml

```
@@ -0,0 +1,300 @@
+---
+id: org.thonny.Thonny
+branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: "20.08"
```

```suggestion
runtime-version: "21.08"
```

21.08 is out and I think it's a good opportunity to use it